beta/src/components/MDX/Sandpack/index.tsx

```diff
@@ -1,8 +1,14 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ */
+
 import * as React from 'react';
-import {reducedCodeSnippet} from './utils';
-const Sandpack = React.lazy(() => import('./SandpackWrapper'));
+import dynamic from 'next/dynamic';
+import {createFileMap} from './utils';
+
+const SandpackRoot = dynamic(() => import('./SandpackRoot'), {suspense: true});
```

Comment:
I switched this to dynamic because iirc this is currently optimized in Next.js (it starts prefetching them early).